content/syllabuses/ios-mobile.md

```diff
@@ -0,0 +1,16 @@
+---
+title: iOS Mobile Development
+---
+
+## Course Content Link
+- {{% contentlink "https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6" %}}
```

Comment:
Should be:
- [Stanford CS193P iOS11 Swift4](https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6)